spec/cassettes/scrape_pdf.yml

```diff
@@ -0,0 +1,1342 @@
+---
```

Comment:
this seems to be a type of configuration file for mocking scraping pdfs instead of making the requests right?